sdk/documentintelligence/Azure.AI.DocumentIntelligence/src/Generated/ClassifyDocumentOptions.Serialization.cs

```diff
@@ -44,13 +44,28 @@ protected virtual void JsonModelWriteCore(Utf8JsonWriter writer, ModelReaderWrit
                 writer.WritePropertyName("base64Source"u8);
                 writer.WriteBase64StringValue(BytesSource.ToArray(), "D");
             }
+            if (Optional.IsDefined(ClassifierId))
```

Comment:
You must be careful when modifying this file. As you can see, it's inside our `Generated` folder. Once we run the code generator again, these changes will be overwritten.

The right way of adding changes to this method would be to move it to the other `ClassifyDocumentOptions` file, located outside of the `Generated` folder ([ClassifyDocumentOptions](https://github.com/Azure/azure-sdk-for-net/blob/4709b9292d9d9c4567a89d5595faea97a6d28355/sdk/documentintelligence/Azure.AI.DocumentIntelligence/src/ClassifyDocumentOptions.cs)). This should prevent a method with the same signature from being generated when we run the generator again.

Reply:
Thanks for revieing.

I am sorry, I am a bit confused here. 

Right now, I introduced this to the `ClassifyDocumentOptions` file located outside of the Generated folder. So when the  code generator runs again it would update create a new `ClassifyDocumentOptions.Serialization.cs` and `JsonModelWriteCore` will have new code?
```
internal ClassifyDocumentOptions(string classifierId,
            Uri uriSource,
            BinaryData bytesSource,
            SplitMode? split,
            string pages,
            IDictionary<string, BinaryData> serializedAdditionalRawData)
        {
            ClassifierId = classifierId;
            UriSource = uriSource;
            BytesSource = bytesSource;
            Split = split;
            Pages = pages;
            _serializedAdditionalRawData = serializedAdditionalRawData;
        }
```